php-plugin/src/main/java/fr/greencodeinitiative/php/PhpRulesDefinition.java

```diff
@@ -39,9 +39,9 @@
 public class PhpRulesDefinition implements RulesDefinition {
```

Comment:
with this review, I realize that there are two files to create rules definition for php plugin ... I will correct it later after your PR merged. I create now an issue for this (#85 )